bugbug/models/accessibility.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import logging
+from datetime import datetime
+
+import xgboost
+from dateutil.relativedelta import relativedelta
+from imblearn.over_sampling import BorderlineSMOTE
+from imblearn.pipeline import Pipeline as ImblearnPipeline
+from sklearn.compose import ColumnTransformer
+from sklearn.feature_extraction import DictVectorizer
+from sklearn.pipeline import Pipeline
+
+from bugbug import bug_features, bugzilla, feature_cleanup, utils
+from bugbug.model import BugModel
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class AccessibilityModel(BugModel):
+    def __init__(self, lemmatization=False):
+        BugModel.__init__(self, lemmatization)
+
+        self.calculate_importance = False
+
+        feature_extractors = [
+            bug_features.HasSTR(),
+            bug_features.Severity(),
```

Comment:
After rolling back, there are quite a number of accessibility bugs that were filed with severity set at `creation_ts` .Adding the feature does improve metrics.

Reply:
Could you give some examples of bugs that were filled with severity at the time of filing?

Reply:
The quickest examples would be:
1. https://bugzilla.mozilla.org/show_bug.cgi?id=1869800
2. https://bugzilla.mozilla.org/show_bug.cgi?id=1860562
3. https://bugzilla.mozilla.org/show_bug.cgi?id=1754097

<details>
<summary>Additional examples below (More than these)</summary>

```
 1591210,
 1628476,
 1783868,
 1790421,
 1799460,
 1812251,
 1843293,
 1843309,
 1843327,
 1843338,
 1843343,
 1844051,
 1844052,
 1844075,
 1844084,
 1849573,
 1849577,
 1849697,
 1852697,
 1857457,
 1857458,
 1858374,
 1868587,
 1560683,
 1561288,
 1561951,
 1565254,
 1565559,
 1565702,
 1565896,
 1566183,
 1566559,
 1566685,
 1567422,
 1567440,
 1568749,
 1568841,
 1569931,
 1569934,
 1570219,
 1570571,
 1570645,
 1570716,
 1570717,
 1571242,
 1571389,
 1571691,
 1572019,
 1572029,
 1572109,
 1572198,
 1572531,
 1572564,
 1572641,
 1573197,
 1574044,
 1574079,
 1574082,
 1574389,
 1574726,
 1574727,
 1574728,
 1574729,
 1574730,
 1574736,
 1574940,
 1574949,
 1575446,
 1575622,
 1575749,
 1575755,
 1576026,
 1576047,
 1576067,
 1576175,
 1576509,
 1576510,
 1576779,
 1577148,
 1577801,
 1578051,
 1578052,
 1578316,
 1578334,
 1578445,
 1578873,
 1579135,
 1579610,
 1580859,
 1581231,
 1582544,
 1583425,
 1583426,
 1583428,
 1584220,
 1584730,
 1584859,
 1585608,
 1585610,
 1585907,
 1585912,
 1587809,
 1587959,
 1588109,
 1589007,
 1589632,
 1589859,
 1589862,
 1590215,
 1590324,
 1590432,
 1591539,
 1592654,
 1592719,
 1593737,
 1593900,
 1594099,
 1594107,
 1594608,
 1595194,
 1596050,
 1597183,
 1597993,
 1598979,
 1599747,
 1601135,
 1601783,
 1602385,
 1604140,
 1605357,
 1605869,
 1606920,
 1606923,
 1606989,
 1607528,
 1608288,
 1608766,
 1608923,
 1609827,
 1610081,
 1610122,
 1610334,
 1610913,
 1611793,
 1612055,
 1613603,
 1614162,
 1614238,
 1614482,
 1614782,
 1614921,
 1614939,
 1615785,
 1615787,
 1616090,
 1616324,
 1616367,
 1616440,
 1616843,
 1616916,
 1617005,
 1617151,
 1617199,
 1617374,
 1617489,
 1617678,
 1617681,
 1618931,
 1619318,
 1619873,
 1620311,
 1620718,
 1620899,
 1622503,
 1622534,
 1622723,
 1622995,
 1623678,
 1624300,
 1626299,
 1626543,
 1626678,
 1626849,
 1628129,
 1628396,
 1629303,
 1629365,
 1629965,
 1631729,
 1632453,
 1633911,
 1633919,
 1635680,
 1636785,
 1637230,
 1637536,
 1637980,
 1640502,
 1700872,
 1675015,
 1643467,
 1678092,
 1668502,
 1648069,
 1702027,
 1644693,
 1644669,
 1666885,
 1652632,
 1685699,
 1648011,
 1661679,
 1644127,
 1644343,
 1810024,
 1706966,
 1709855,
 1771802,
 1703739,
 1752051,
 1763362,
 1735647,
 1799714,
 1752028,
 1733438,
 1772016,
 1769957,
 1711258,
 1716344,
 1751978,
 1754097,
 1707659,
 1795360,
 1817085,
 1817109,
 1818086,
 1829095,
 1829099,
 1853226,
 1854042,
 1854356,
 1854607,
 1854787,
 1855430,
 1856396,
 1858822,
 1858824,
 1858825,
 1858986,
 1859894,
 1860272,
 1860562,
 1861019,
 1861568,
 1868863,
 1868905,
 1869800,
 1870299

```
</details>